.github/workflows/docs.yaml

```diff
@@ -55,7 +55,7 @@ jobs:
           # the wheel unless the label cliflow/binaries/all is present in the
           # PR.
         python-version: ['3.10']
-        cuda-version: ['12.6']
+        cuda-version: ['12.8']
```

Comment:
Drive-by, 12.6 is still supported but the "default" CUDA version supported by pytorch is currently 12.8